yaml/stopwords_id.yml

```diff
@@ -0,0 +1,66 @@
+# Created by Hiroko Kinoshita for the marimo stopwords: https://github.com/koheiw/marimo
+
+pronoun: 
+    basic: 
+        # [i, me, myself, we, ours, ourselves, you, yours, yourself, yourselves, he, him, himself, she, hers, herself]
+        - [saya, aku, gue, sendiri, kita, kami, anda, kamu, engkau, saudara, milikmu, dirimu, dia, ia, beliau, kepunyaannya, punya*]
+        # [it, itself, they, them, theirs, themselves, this, that, these, those]
+        - [itu, ini, mereka, bahwa*, yang]
+    possesive: 
+        # [my, our, your, his, her, its, their][my, our, your, his, her, its, their]
+        - [milikku, milik saya, milik*]
+    interogative: 
+        # [what, which, who, whom, whose, when, where, why, how]
+        - [apa, mana, siapa*, kapan, ketika, mengapa, kenapa, bagaimana]
+    contraction: 
+        # [i'm, you're, he's, she's, it's, we're, they're, i've, you've, we've, they've, i'd, you'd, he'd, she'd, we'd, they'd, i'll, you'll, he'll, she'll, we'll, they'll]
+    
+verb: 
+    basic: 
+        # [am, is, are, was, were, be, been, being, have, has, had, having, do, does, did, doing]
+        - [adalah, dulu, menjadi,berada, telah, makhluk, melakukan, berbuat, dilakukan, perbuatan]
+    modal: 
+        # [would, should, could, ought, will]
+        - [akan, harus, bisa, seharusnya]
+    contraction: 
+        # [isn't, aren't, wasn't, weren't, hasn't, haven't, hadn't, doesn't, don't, didn't, won't, wouldn't, shan't, shouldn't, can't, cannot, couldn't, mustn't, let's, that's, who's, what's, here's, there's, when's, where's, why's, how's]
+    
+    reporting: 
+        # [say, says, said, tell, tells, told]
+        - [mengatakan, kata*, berkata, mengucapkan, berucap, ujar, berpendapat, bilang, membilangkan, mengomong, memprikan,mengira, menceritakan, menyuruh, bercerita, mengetahui, mengenal*, menunjukkan]
+        # [report, reports, reported]
+        - [jelaskan, deberitahu, melapor*, mengabarkan, memberitahu*, mewartakan, lapor*, merencana, repot, dilaporkan]
+article: 
+    # [a, an, the]
+    - [sebuah, sebatang, sang]
+conjunction:
+    # [and, but, if, or, because, so, while, nor]
+    - [dan, serta, dengan, tambah, tetapi, melainkan, jika, atau, karena, sebab, begitu, amat, jadi, demikian, sementara, sedangkan, sambil, meski*, walaupun, maupun, namun]
+adverb: 
+    # [as, until, once, here, there, all, any, both, each, few, many, more, most]
+    - [sebagai, seperti, ajak, sampai, sehingga, hingga, sekali, sini, sana, semua, segenap, segela, sekalian, sesuatu, kedua*, tiap*, masing*, banyak, lebih banyak]
```

Comment:
Can we separte "lebih banyak"?

Reply:
I don't think we can, because Indonesian language doesn't have single comparative.

Reply:
Is "lebih" or "banyak" part of other important expressions?